AUTHORS

```diff
@@ -1,11 +1,46 @@
-## OpenCRVS Lead Architects:
+## OpenCRVS Contributing organisations:
+
+OpenCRVS Limited
+Plan International
+Vital Strategies
+Jembi Health Systems
+
+## OpenCRVS Code Contributors:
 
 - Euan Millar https://github.com/euanmillar
-- Ryan Crichton https://github.com/rcrichton
 - Riku Rouvila https://github.com/rikukissa
+- Tameem Bin Haider https://github.com/Zangetsu101
+- Tahmid Rahman https://github.com/tahmidrahman-dsi
+- Pyry Rouvila https://github.com/naftis
+- Ashikul Alam https://github.com/Nil20
+- Jonathan Pye-Finch https://github.com/jpye-finch
+- Sadman Anik https://github.com/sadmananik
 - Mushraful Hoque Anik https://github.com/mushrafulhoque-dsi
-
-Plus many. many more valued contributors all listed <a href="https://github.com/opencrvs/opencrvs-core/graphs/contributors">here</a>
+- Ryan Crichton https://github.com/rcrichton
+- Atiq Zaman https://github.com/atiqzaman-dsi
+- Kayumuzzaman ("Robin") https://github.com/kayumuzzaman
+- Rabiul Islam Anik https://github.com/rabiulislamanik
+- Mahfuzur Rahman Emon https://github.com/emon-swe-sust
+- Tomi Salo https://github.com/Eezi
```

Comment:
@Eezi THANK YOU!!! :heart_eyes: ... for you contribution to ensure every individual on the planet is recognised, protected and provided for from birth!